content/exporters/supported-exporters/Node.js/Prometheus.md

```diff
@@ -31,26 +34,56 @@ npm install @opencensus/exporter-prometheus
 ```
 
 ## Creating the exporter
+To create the exporter, we'll need to:
+
+* Import and use the Prometheus exporter package
+* Define a namespace that will uniquely identify our metrics when viewed on Prometheus
+* Expose a port on which we shall run a `/metrics` endpoint
+* With the defined port, we'll need a Promethus configuration file so that Prometheus can scrape from this endpoint
+
 Now let's use the Prometheus exporter:
 
 ```js
-const { Stats } = require('@opencensus/core');
+const { globalStats } = require('@opencensus/core');
 const { PrometheusStatsExporter } = require('@opencensus/exporter-prometheus');
 
 // Add your port and startServer to the Prometheus options
 const exporter = new PrometheusStatsExporter({
   port: 9464,
-  startServer: false
+  startServer: true
```

Comment:
What changed here from `startServer: false` to `startServer: true`?

Reply:
So earlier, it was like two step approach

1. Don't start the Prometheus exporter  server i.e. ```startServer: false```
2. Explicitly call ```startServer``` method to start the Prometheus exporter server i.e.
```// Run the server
exporter.startServer(function callback() {
  // Callback
});
```

This can be possible by passing ```startServer: true``` option.